Source/WebKit/NetworkProcess/NetworkSession.cpp

```diff
@@ -496,6 +496,12 @@ void NetworkSession::firePrivateClickMeasurementTimerImmediatelyForTesting()
     privateClickMeasurement().startTimerImmediatelyForTesting();
 }
 
+void NetworkSession::setBlobRegistryTopOriginPartitioningEnabled(bool enabled)
+{
+    RELEASE_LOG(Storage, "NetworkSession::setBlobRegistryTopOriginPartitioningEnabled as %s", enabled ? "enabled" : "disabled");
```

Comment:
You shouldn't use `%s` in RELEASE_LOG as it will get stripped. You'll need to use:
```
RELEASE_LOG(Storage, "NetworkSession::setBlobRegistryTopOriginPartitioningEnabled as %" PUBLIC_LOG_STRING, enabled ? "enabled" : "disabled");
```